.gitignore

```diff
@@ -30,3 +30,4 @@ target/
 .*.swp
 .swo
 .*.swo
+lowered.hnir
```

Comment:
This file is generated on `coreNative/test`. I think it no longer does this in the upcoming native version